from0k2bp.tex

```diff
@@ -185,16 +185,16 @@
 Zero Knowledge Proofs generally.
 
 More importantly though, this is for now just a personal investigation
-and is bound to contain both gross and subtle errors. Corrections will
-be welcome, but readers should bear this in mind.
+and is bound to contain both gross and subtle errors. Corrections are welcomed, 
+but readers should bear this in mind.
 
 \hypertarget{notation}{%
 \subsection[Notation]{\texorpdfstring{\protect\hypertarget{anchor-11}{}{}Notation}{Notation}}\label{notation}}
 
 We will be working in the elliptic curve scenario, assuming everywhere
 the use of a curve for which the standard generator/basepoint is $G$, and
-whose order is $p$(NB; this is usually $N$, but we avoid that because is used
-as a vector dimension often).
+whose order is~$p$\footnote{Nota bene, this is usually $N$, but we avoid that notation because 
```

Comment:
NB is standard for `Nota bene`, in fact you almost never see the full form like that. Perhaps "N.B." is correct rather than "NB", but I mean I'm not trying to adhere to some kind of style guide here :)

Reply:
It took me a few seconds of thinking what that NB stands for. I've read several papers but as far as I remember `NB` was not used there. So this represents only my inexperience I guess. I'll revert it.